clients/client/src/main/java/org/projectnessie/client/builder/ClientSideCreateNamespace.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.client.builder;
+
+import java.util.Map;
+import java.util.Optional;
+import org.projectnessie.client.api.NessieApiV2;
+import org.projectnessie.error.NessieConflictException;
+import org.projectnessie.error.NessieNamespaceAlreadyExistsException;
+import org.projectnessie.error.NessieNotFoundException;
+import org.projectnessie.error.NessieReferenceNotFoundException;
+import org.projectnessie.model.Branch;
+import org.projectnessie.model.CommitMeta;
+import org.projectnessie.model.Content;
+import org.projectnessie.model.ContentKey;
+import org.projectnessie.model.ImmutableNamespace;
+import org.projectnessie.model.Namespace;
+import org.projectnessie.model.Operation;
+
+/** Supports previous "create namespace" functionality of the java client over Nessie API v2. */
+public final class ClientSideCreateNamespace extends BaseCreateNamespaceBuilder {
```

Comment:
It not specific to v2, could be used with v1 or another other API version that has "get entries" and "commit".

I put it in the `builder` package because it implemented the `CreateNamespaceBuilder` interface. However, the logic in this class is indeed different from other builder...

How does `...client.helper.ClientSideCreateNamespace` sound?... or `client.util`?.. Any other package name suggestions?

Reply:
Moved to `client.util.v2api` and added javadoc.